assignments/3. type conversion/index.md

```diff
@@ -1,36 +1,36 @@
-1. 🎖 What's the outpit of the code below
+1. 🎖 What's the output of the code below
 ```js
-typeof "Joe";
-typeof 4;
-typeof NaN;
-typeof false;
-typeof function () {};
-var phone = 8983700;
-typeof phone;
-typeof null;
+typeof "Joe"; string
+typeof 4; number
+typeof NaN; number
+typeof false; boolean
+typeof function () {}; function
+var phone = 8983700; 
+typeof phone; number
+typeof null; object
```

Comment:
`typeof` returns string kind of data. It should be `"string"` not `string` same for others.